easybuild/easyconfigs/p/Python/Python-2.7.9-gompi-1.5.16.eb

```diff
@@ -0,0 +1,24 @@
+name = 'Python'
+version = '2.7.9'
+
```

Comment:
no extensions included, so make this a `-bare` version?

see https://github.com/hpcugent/easybuild-easyconfigs/blob/master/easybuild/easyconfigs/p/Python/Python-2.5.6-goolf-1.4.10-bare.eb